bitnami/superset/templates/_helpers.tpl

```diff
@@ -0,0 +1,397 @@
+    {{ default (include "common.names.fullname" .) .Values.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Get the Redis&reg; credentials secret.
+*/}}
+{{- define "superset.redis.secretName" -}}
+{{- if .Values.redis.enabled -}}
+    {{- $name := default "redis" .Values.redis.nameOverride -}}
+    {{- default (printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-") (tpl .Values.redis.auth.existingSecret $) -}}
+{{- else }}
+    {{- default (printf "%s-externalredis" .Release.Name) (tpl .Values.externalRedis.existingSecret $) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Get the Postgresql credentials secret.
+*/}}
+{{- define "superset.postgresql.secretName" -}}
+{{- if .Values.postgresql.enabled }}
+    {{- tpl (coalesce (((.Values.global).postgresql).auth).existingSecret .Values.postgresql.auth.existingSecret (include "superset.postgresql.fullname" .)) $ -}}
+{{- else -}}
+    {{- default (printf "%s-externaldb" .Release.Name) (tpl .Values.externalDatabase.existingSecret $) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Get the secret name
+*/}}
+{{- define "superset.secretName" -}}
+{{- default (include "common.names.fullname" .) (tpl .Values.auth.existingSecret .) -}}
+{{- end -}}
+
+{{/*
+Get the configmap name
+*/}}
+{{- define "superset.configMapName" -}}
+{{- default (printf "%s-configuration" (include "common.names.fullname" .)) (tpl .Values.existingConfigmap .) -}}
+{{- end -}}
+
+{{/*
+Add environment variables to configure database values
+*/}}
+{{- define "superset.database.host" -}}
+{{- ternary (include "superset.postgresql.fullname" .) .Values.externalDatabase.host .Values.postgresql.enabled -}}
+{{- end -}}
+
+{{/*
+Add environment variables to configure database values
+*/}}
+{{- define "superset.database.user" -}}
+{{- if .Values.postgresql.enabled }}
+    {{- default .Values.postgresql.auth.username (((.Values.global).postgresql).auth).username -}}
+{{- else -}}
+    {{- .Values.externalDatabase.user -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Add environment variables to configure database values
+*/}}
+{{- define "superset.database.name" -}}
+{{- if .Values.postgresql.enabled }}
+    {{- default .Values.postgresql.auth.database (((.Values.global).postgresql).auth).database -}}
+{{- else -}}
+    {{- .Values.externalDatabase.database -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Add environment variables to configure database values
+*/}}
+{{- define "superset.database.secretKey" -}}
+{{- if .Values.postgresql.enabled -}}
+    {{- print "password" -}}
+{{- else -}}
+    {{- print .Values.externalDatabase.existingSecretPasswordKey -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Add environment variables to configure database values
+*/}}
+{{- define "superset.database.port" -}}
+{{- if .Values.postgresql.enabled -}}
+    {{- default .Values.postgresql.primary.service.ports.postgresql ((((.Values.global).postgresql).service).ports).postgresql -}}
```

Comment:
I would add `postgresql.primary.service.ports.postgresql` and `redis.master.service.ports.redis` to the values.yaml file.